app/main/lib/similarity.py

```diff
@@ -93,6 +106,12 @@ def add_item(item, similarity_type):
   app.logger.info(f"[Alegre Similarity] [Item {item}, Similarity type: {similarity_type}] response for add was {response}")
   return response
 
+def callback_add_item(item, similarity_type):
+  if similarity_type == "audio":
+      response = audio_model().add(item)
+  app.logger.info(f"[Alegre Similarity] CallbackAddItem: [Item {item}, Similarity type: {similarity_type}] Response looks like {response}")
```

Comment:
debug?

Reply:
Let's also add an `else` before this line with a `app.warn` or `app.error` statement if `similarity_type` is anything other than `audio` for now.